build/toolchain/android/BUILD.gn

```diff
@@ -178,6 +178,10 @@ android_gcc_toolchains_helper("arm64") {
   toolchain_cpu = "aarch64"
 }
 
+# This toolchain should only be used to build the target
+# //third_party/vulkan_validation_layers. This is because vulkan validation
+# layers requires API level >= 26, but Flutter officially supports down to API
+# level 22, which is the default value of the android_api_level argument.
```

Comment:
might want to mention something about how this is safe only because of runtime checks before loading the dynamic library

Reply:
Yeah good point. Adding a comment to that effect in https://github.com/flutter/engine/pull/45467.